queue_services/entity-emailer/requirements.txt

```diff
@@ -78,6 +78,6 @@ webcolors==1.13
 Werkzeug==1.0.1
 yarl==1.8.2
 zipp==3.15.0
-git+https://github.com/bcgov/business-schemas.git@2.18.10#egg=registry_schemas
-git+https://github.com/seeker25/lear.git@16466___#egg=legal_api&subdirectory=legal-api
+git+https://github.com/bcgov/business-schemas.git@2.18.14#egg=registry_schemas
+git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
 git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
```

Comment:
Not sure if it was intentional to install legal-api from Travis' PR, but the rest of the code imports it from the bcgov repo

Reply:
thanks for catching this!